Library/Formula/postgresql.rb

```diff
@@ -1,7 +1,6 @@
 class Postgresql < Formula
   desc "Object-relational database system"
   homepage "https://www.postgresql.org/"
-  revision 2
```

Comment:
I know the documentation merged the other day isn't explicit about that; I'll fix that in the morning.